### ISSUE TITLE
Extended integers ("Long" type) 

### ISSUE THREAD

**Issue by dusty-phillips** [chrivers/pyjaco#22]
This one would be redundant in Python 3

**Comment by chrivers** [comment]
Not exactly. JavaScript has no support for long integers at all, so no matter if it's a separate pyjaco type or not, we need to make an implementation (or use an existing one) of long integers in javascript. They are typically implemented via strings or arrays of integers.
